src/test/java/uk/gov/justice/digital/hmpps/keyworker/integration/health/InfoIntTest.kt

```diff
@@ -0,0 +1,87 @@
+package uk.gov.justice.digital.hmpps.keyworker.integration.health
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.keyworker.integration.IntegrationTest
+import uk.gov.justice.digital.hmpps.keyworker.integration.KeyworkerServiceIntegrationTest.Companion.KEYWORKER_ID_1
+import uk.gov.justice.digital.hmpps.keyworker.integration.KeyworkerServiceIntegrationTest.Companion.MIGRATED_ALLOCATION_OFFENDER_ID
+import uk.gov.justice.digital.hmpps.keyworker.integration.KeyworkerServiceIntegrationTest.Companion.NON_MIGRATED_ALLOCATION_OFFENDER_ID
+import uk.gov.justice.digital.hmpps.keyworker.integration.KeyworkerServiceIntegrationTest.Companion.PRISON_ID
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter.ISO_DATE
+import java.util.function.Consumer
+
+class InfoIntTest : IntegrationTest
+() {
+  @Test
+  fun `Info page contains git information`() {
+    webTestClient.get().uri("/info")
+      .exchange()
+      .expectStatus().isOk
+      .expectBody().jsonPath("git.commit.id").isNotEmpty
+  }
+
+  @Test
+  fun `Info page reports version`() {
+    webTestClient.get().uri("/info")
+      .exchange()
+      .expectStatus().isOk
+      .expectBody().jsonPath("build.version").value(
+        Consumer<String> {
+          assertThat(it).startsWith(LocalDateTime.now().format(ISO_DATE))
+        }
+      )
+  }
+
+  @Test
+  fun `Info page still works when items in ehcache`() {
+    migratedFoAutoAllocation(PRISON_ID)
+    eliteMockServer.stubOffendersAtLocationForAutoAllocation(
+      getWiremockResponse(PRISON_ID, "offenders-at-location")
+    )
+    eliteMockServer.stubKeyworkerRoles(
+      PRISON_ID,
+      KEYWORKER_ID_1,
+      getWiremockResponse(PRISON_ID, "staff-location-role-list")
+    )
+    addKeyworkerAllocation(PRISON_ID, NON_MIGRATED_ALLOCATION_OFFENDER_ID)
+    eliteMockServer.stubkeyworkerDetails(KEYWORKER_ID_1, getWiremockResponse("staff-details--5"))
+    eliteMockServer.stubOffendersAllocationHistory(getWiremockResponse(PRISON_ID, "offenders-history"))
+    eliteMockServer.stubPrisonerStatus(
+      NON_MIGRATED_ALLOCATION_OFFENDER_ID,
+      getWiremockResponse("prisoners_information_A1234AA")
+    )
+
+    webTestClient.get()
+      .uri("/key-worker/allocation-history/$NON_MIGRATED_ALLOCATION_OFFENDER_ID")
+      .headers(setOmicAdminHeaders())
+      .exchange()
+      .expectStatus().is2xxSuccessful
+
+    webTestClient.get().uri("/info")
```

Comment:
annoyingly the `/info` endpoint works fine when there isn't anything in the cache so need :point_up: to populate it first